### DISCUSSION TITLE
PrimeNG - Não consigo alinhar menu items na direita

### DISCUSSION THREAD

**Discussion by Jotta-art** [frontendbr/forum#2334] [Perguntas e Respostas]
Estou usando a última versão do PrimeNG e estou usando um menubar. Não estou conseguindo alinhar seus itens a direita.

Estou tentando alinhar os seguintes itens a direita:

![image](https://user-images.githubusercontent.com/61245734/184652186-a49d3d74-19de-4ff3-acb3-1a2ed098c01e.png)

**Comment by felquis** [comment]
Alinhar itens a direita.

Se os itens são elementos, daria para fazer com css: `display: flex; align-items: flex-end;` 

Se os itens são apenas texto, daria para fazer com css: `text-align: right;`

Veja se te ajuda, se não ajudar, tenta mandar para nós como é o DOM diretamente no DevTools, pois para conseguir alinhar elementos com CSS, é necessário entender bem como é a estrutura do HTML em volta do elemento que precisa ser alinhado, e ai partir para o CSS para posiciona-los. Bele?

**Reply by Jotta-art** [reply]
Eu já tinha tentado esses e nada. Vou mandar uma foto do DOM aqui.

![image](https://user-images.githubusercontent.com/61245734/185167654-20c71633-f2c9-4490-9890-b9149c0ff9ca.png)

![image](https://user-images.githubusercontent.com/61245734/185168204-989fd56c-8875-44e0-bb25-da8b11ab308e.png)
